wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -1250,4 +1248,25 @@ private void registerCrowdNodeConfirmedAddressFilter() {
             apiConfirmationHandler = null;
         }
     }
+
+    InputStream loadStream(String filename) {
+        InputStream stream = null;
+        try {
+            stream = getAssets().open(filename);
+        } catch (IOException x) {
+            log.warn("cannot load the bootstrap stream: {}", x.getMessage());
```

Comment:
it is OK if a bootstrap file is missing.  Typically for devnet builds, we don't have such files.  For mainnet, we should have them.